### PR TITLE
FIX: SoundFile-cue everytime return empty Event object

### PR DESCRIPTION
**Pull request by nilninull**
SoundFile-cue method makes a empty Event object and it can't control the playing status.

I changed the code in the fork block to use the object reference.

### CODE REVIEW

File: SCClassLibrary/Common/Files/SoundFile.sc [hunk 1]
```diff
@@ -407,10 +407,10 @@ SoundFile {
 
 	cue { | ev, playNow = false, closeWhenDone = false |
 		var server, packet, defname = "diskIn" ++ numChannels, condition, onClose;
-		ev = ev ? ();
+		ev = ev !? `_ ? `();
```
Comment (telephon) [inline SCClassLibrary/Common/Files/SoundFile.sc]:
```
(
f =  { |ev|
	ev !? `_ ? `();
	ev
}
)

f.(); // returns nil
f.( (a: 3));  // returns  ( 'a': 3 )

// none of them return a Ref.
```

did you mean to write:
```
ev = reference(ev ? ());
```

File: SCClassLibrary/Common/Files/SoundFile.sc [hunk 2]
```diff
@@ -440,7 +440,7 @@ SoundFile {
 					}, "/n_end", server.addr, nil, ev[\id][0]).oneShot;
 				};
 				if (playNow) {
-					packet = server.makeBundle(false, {ev.play})[0];
+					packet = server.makeBundle(false, {ev.value.play})[0];
```
Comment (telephon) [inline SCClassLibrary/Common/Files/SoundFile.sc]: The first argument to `makeBundle` is time: the old `false` was wrong.

While you are at it: minor formatting thing - `{ ev.value.play }` (add spaces).

File: SCClassLibrary/Common/Files/SoundFile.sc (hunk 1 above)
Comment (telephon) [inline SCClassLibrary/Common/Files/SoundFile.sc]: I don't think that you really need a `Ref`. The event is already an object you can refer to.

File: SCClassLibrary/Common/Files/SoundFile.sc (hunk 2 above)
Comment (nilninull) [inline SCClassLibrary/Common/Files/SoundFile.sc]: I'm sorry, I was wrong.

In my (old?) environment, when I used the SoundFile-cue method, I always received an empty Event object. And it was fixed by changing the object to reference.

But now, I rechecked the code, SoundFile-cue method returns playable object.

So I cancel my pull request.

ps.
Thanks for your codes.

`ev = reference(ev ? ());`

This is a more sophisticated one of my intended code.

And this code, I wrote it for the purpose of creating a reference to the Event object.
```
(
f =  { |ev|
	ev = ev !? `_ ? `();
	ev
}
)

f.(); // returns `((  )) 
f.( (a: 3));  // returns `(( 'a': 3 ))
```

Comment (telephon) [inline SCClassLibrary/Common/Files/SoundFile.sc]: ok! I must have miswritten it somehow, sorry for that.

**Pull request closed without merging**